src/pybind/mgr/dashboard/frontend/src/app/ceph/block/mirroring/daemon-list/daemon-list.component.html

```diff
@@ -9,5 +9,5 @@
 <ng-template #healthTmpl
              let-row="data.row"
              let-value="data.value">
-  <span [ngClass]="row.health_color | mirrorHealthColor">{{ value }}</span>
+  <cds-tag [class]="row.health_color | mirrorHealthColor">{{ value }}</cds-tag>
```

Comment:
Smae question here, why are we replacing `ngClass` with `class`

Reply:
![image](https://github.com/user-attachments/assets/64038eab-499e-46de-a5e7-9679c4859967)
 
In input, it supports only classes in Cds-tag. I have tried ngclass as well, but it will return, string(classname) and CSS changes, not reflect